code/src/components/Congrats.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import Lottie from "react-lottie"
+import animationData from "../lotties/congrats"
+import { StyledCongrats } from "styles"
+
+const defaultOptions = {
+  loop: true,
+  autoplay: true,
+  animationData,
+  renderSettings: {
+    preserveAspectRation: "xMidYMid slice",
```

Comment:
Nice with the lottie! Havn't tried that out yet, i'm to scared haha. But your animation looks amazing!